webdriver/src/com/blazemeter/jmeter/webdriver/phantomjs/PhantomJSDriverConfig.java

```diff
@@ -34,6 +36,13 @@ protected Capabilities createCapabilities() {
         capabilities.setCapability(CapabilityType.PROXY, createProxy());
         capabilities.setCapability(PhantomJSDriverService.PHANTOMJS_EXECUTABLE_PATH_PROPERTY,
                 getPhantomJsExecutablePath());
+        String cliArgs = JMeterContextService.getContext().getVariables().get("phantomjs.cli.args");

```

Comment:
I chose that way because - it is the easiest way for me to reach my goal - start jmeter and set up some "global" variables, among them I would pass cli args for phantomjs driver. Moreover I'd like to add some options for firefox, for example set a Xdisplay to use. (I want to start firefox on xvfb's display, not on primary X display in Ubuntu). And of course I can pass custom cli arg to phantomjs through passing that variable to JMeter's cli. For me it is very useful. And I would glad to know why  I am wrong.